cpp/tests/quantiles/quantiles_test.cu

```diff
@@ -33,97 +34,53 @@ using namespace cudf;
 using namespace test;
 
 template <typename T>
-struct QuantilesTest : public BaseFixture {
+struct DiscreteQuantilesTest : public BaseFixture {
```

Comment:
I think there need to be tests of the exceptional cases (make sure things that should throw do throw)?

Reply:
I'll add more to account for the API changes. :+1: